README.md

```diff
@@ -4,21 +4,35 @@
 
 Before you can run the site, make sure you have [asciidoc](http://www.methods.co.nz/asciidoc/) installed on your machine!
 
-## Building
+## Running locally
 
-1. Install rvm (https://rvm.io/)
-1. Run `rvm install ruby 2.2.2`
-1. Run `rvm gemset create aerogear`
-1. Run `rvm use ruby-2.2.2@aerogear`
-1. Run `gem install bundle --no-ri --no-rdoc`
-1. Run `bundle install --path vendor` to install the dependencies
-1. Run `bundle exec jekyll serve --watch`
-1. go to <http://127.0.0.1:4000/>
+Ruby & its ecosystem is difficult to install and manage, so we recommend running Jekyll & aerogear.org in via Docker.
 
-### Development Steps
+### Building
+
+Build and tag a local docker image via:
+
+```
+[sudo] docker build --tag aerogear.org .
+```
+
+*Note*: This will clone a fresh copy of the `aerogear.org` repo into the container.
```

Comment:
Related to comment https://github.com/aerogear/aerogear.org/pull/725/files#r185816918
Why do a fresh clone? why not mount the already cloned src on the host

Reply:
OK. Not a reason to block the merge